components/ShareCredentials.vue

```diff
@@ -166,11 +166,12 @@ export default {
         });
         // FIXME: use matches directly instead of using records, UI
         // should better render full matches
-        records = matches.flat.map(match => match.derived?.length > 0 ?
+        records = Array.isArray(matches.flat) ?
+          matches.flat.map(match => match.derived?.length > 0 ?
```

Comment:
Reverting this since it is based on a bug in bedrock-web-wallet that has been fixed:

```suggestion
        records = matches.flat.map(match => match.derived?.length > 0 ?
```